### PR TITLE
Planning Reviews Pt2

### PR DESCRIPTION
**Pull request by dhaselhan**
* Add overview component, detailed header
* Add updating the type of the PR
* Add staff journal and legacy ID
* Add noi prefix to roso, pfrs components to remove warnings about overlapping tags

### CODE REVIEW

**Review by mhuseinov — APPROVED** [review]
👍

File: alcs-frontend/src/app/features/planning-review/header/header.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { HeaderComponent } from './header.component';
+
+describe('DetailsHeaderComponent', () => {
```
Comment (sandratoh) [inline alcs-frontend/src/app/features/planning-review/header/header.component.spec.ts:6]: Should this not match the name of the component 'HeaderComponent'?

File: services/apps/alcs/src/alcs/staff-journal/staff-journal.dto.ts [hunk 2]
```diff
@@ -40,14 +42,16 @@ export class CreateNoticeOfIntentStaffJournalDto {
   body: string;
```
Comment (sandratoh) [inline services/apps/alcs/src/alcs/staff-journal/staff-journal.dto.ts:42]: Did you mean to remove `body` after extending base?

**Review by sandratoh — APPROVED** [review]
Had some cleanup comments but LGTM 👍

File: services/apps/alcs/src/alcs/staff-journal/staff-journal.dto.ts (hunk 2 above)
Comment (dhaselhan) [reply services/apps/alcs/src/alcs/staff-journal/staff-journal.dto.ts]: Yes, fixed

File: alcs-frontend/src/app/features/planning-review/header/header.component.spec.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/planning-review/header/header.component.spec.ts]: Yes it should, good catch. Fixed.

**Pull request merged**